packages/inventory-next/src/utils/build-query.ts

```diff
@@ -0,0 +1,166 @@
+import { ExtendedFindConfig, FindConfig } from "@medusajs/types"
```

Comment:
question: is this necessary now? shouldn't the build query utils used internally be enough?

Reply:
also it seems more typeorm related right?